data/projects/enhancing_coala_quickstart.md

```diff
@@ -0,0 +1,37 @@
+coala-quickstart is a tool that helps users to quickly get started with coala. It generates a `.coafile` (coala's configuration file) that is tailored to the project. Currently, coala-quickstart automatically identifies relevant bears for the project based on the languages used and has a clean and simple command-line interface for taking user’s input when required.
+
+However, there’s a scope for improvement in the following areas:
+
+- Utilizing information from common configuration files: Most of the projects have configuration files like `Gruntfile.js`, `Gulpfile.js`, `rakefile`, `package.json`, etc and they may contain some settings for executables that `coala-bears` supports. For example, if any project's `Gulpfile.js` has include all files except `somefile.css` for `csslint`, it would be nice if our generated `.coafile` is able to include `CSSLintBear` and reflects this change as well.
+- Improving the interface: 
+	- Fixing warnings: Currently, there are lots of warnings which is not user-friendly.
+	- Reducing typing effort of the user: Currently, the user has to input a lot of values corresponding to the settings which may even occur multiple times in different sections. Example, `SpaceConsistencyBear`.
+	- Validating user input: Currently, whatever values the user enters for the prompted settings gets written into the generated `.coafile`.
+- Continuous Integration: Like other coala repositories, it would be nice to have Appveyor and Travis CI integrations in coala-quickstart.
+
+The aim of the project is to improve the user-experience, extract useful information from the projects to enhance the generated `.coafile` and to improve maintainablility of the codebase. Also, in addition to the above changes the applicants are encouraged to come up with their own ideas.
```

Comment:
This para is kinda redundant when you have prefaced it with so much information above it :)

But keep the last line.